ansible/inventory/offline/99-static

```diff
@@ -0,0 +1,145 @@
+# In this section, add all machines in this installation.
+#
+# Ansible connects to the machine on `ansible_host`
+#
+# The machines talk to eachother on `ip`
+#
+# !!! if `ip` is not provided, ansible will default to the IP of the default
+# interface. Which is probably not what you want
+#
+# <hostname> ansible_host=<internal_ip>
+[all]
+# kubenode1 ansible_host=100.89.110.8  ip=10.114.0.10
+# kubenode2 ansible_host=100.154.219.107 ip=10.114.0.8
+# kubenode3 ansible_host=100.227.143.169 ip=10.114.0.2
+# You could add more if capacity is needed
+# kubenode4 ....
+
+# restund1 ansible_host=XXXX
+# restund2 ansible_host=XXXX
+
+# cassandra1 ansible_host=XXXX
+# cassandra2 ansible_host=XXXX
+# cassandra3 ansible_host=XXXX
+
+# elasticsearch1 ansible_host=XXXX
+# elasticsearch2 ansible_host=XXXX
+# elasticsearch3 ansible_host=XXXX
+#
+# minio1 ansible_host=XXXX
+# minio2 ansible_host=XXXX
+# minio3 ansible_host=XXXX
+#
+# If you are in an offline environment, add an assethost here, from which
+# artifacts are served
+# assethost ansible_host=100.89.14.74 ip=10.114.0.9
+
+# If you need to tunnel ssh connections through a bastion host (because your
+# nodes are not directly reachable from the machine running ansible), define a
+# bastion host as well, and uncomment the [bastion] section below.
+#
+# bastion ansible_host=XXXX ansible_user=some_user
+
+
+# Below variables are set for all machines in the inventory.
+[all:vars]
+# If you need to ssh as a user that's not the same user as the one running ansible
+# set ansible_user=<some_user>
+# Keep in mind this user needs to be able to sudo passwordless.
+# ansible_user = root
+#
+# Usually, you want to have a separate keypair to ssh to these boxes,
+# and tell ansible where it is by setting `ansible_ssh_private_key_file`.
+# ansible_ssh_private_key_file = ./dot_ssh/id_ed25519
+# Note adding it to the ssh agent won't work in scenarios where ansible
+# execution is wrapped through a container, as the ssh-agent socket isn't
+# accessible there.
+
+# Uncomment the following lines when you have both a public and private network
+# and you want your databases to use the private one.
```

Comment:
I think this needs a bit more explanation:
 - Your databases might be expanded to "your cassandra, elastic search and minio instances".
 - This is the recommended network topology no? Also the public/private terminology in an air gapped install might not be the most easy to understand? Not sure what to use instead though...
 - Also `enp1s0` might need to be changed to the name of the private interface, so we want to specify "uncomment and replace the interface name with the name of the private interface on your machines".